### PR TITLE
Update docs for full working nginx auth_request example

### PR DESCRIPTION
**Pull request by semenko**
Addresses @ploxiln comment, closes issue #226.

Working & tested -- though auth_request is very limited without PR #247 and PR #173

Signed-off-by: Nick Semenkovich <semenko@alum.mit.edu>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -355,9 +355,19 @@ server {
...
   }
 
+  location /oauth2/ {
+    proxy_pass http://127.0.0.1:4180;
+    proxy_set_header Host $host;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Scheme $scheme;
+    proxy_connect_timeout 1;
+    proxy_send_timeout 30;
+    proxy_read_timeout 30;
```
Comment (ploxiln) [inline README.md]: I think the timeouts are not necessary for a minimal working example

Comment (semenko) [inline README.md]: Updated!

**Comment by ploxiln** [comment]
👍

**Comment by jehiah** [comment]
thanks @semenko @ploxiln !

**Pull request merged**